dynamiq/nodes/agents/react.py

```diff
@@ -245,8 +270,9 @@ class ReActAgent(Agent):
     """Agent that uses the ReAct strategy for processing tasks by interacting with tools in a loop."""
 
     name: str = "React"
-    max_loops: int = Field(default=15, ge=1)
+    max_loops: int = Field(default=15, ge=2)
     inference_mode: InferenceMode = InferenceMode.DEFAULT
+    raise_on_max_loops: bool = Field(default=True, description="Raise exception when max loops are exceeded.")
```

Comment:
Is there a better name? I cannot say that I like it, but at the same time, I don't have a better suggestion.

@vitalii-dynamiq @acoola, any ideas?

Reply:
Great idea!